File: Eldar Dautović/ridewithme/ridewithme.Model/Models/Recenzija.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ridewithme.Model.Models
{
    public class Recenzija
    {
        public int Id { get; set; }
        public int Ocjena { get; set; }
        public string? Komentar { get; set; }
        public virtual Korisnici Korisnik { get; set; }
        public virtual Voznje Voznja { get; set; }
        public DateTime DatumKreiranja { get; set; }
    }
}

File: Eldar Dautović/ridewithme/UI/ridewithme_admin/lib/models/recenzija.dart
// -- recenzija.dart --
import 'package:json_annotation/json_annotation.dart';
import 'package:ridewithme_admin/models/korisnik.dart';
import 'package:ridewithme_admin/models/voznja.dart';

part 'recenzija.g.dart';

@JsonSerializable()
class Recenzija {
  int? id;
  int? ocjena;
  String? komentar;
  Korisnik? korisnik;
  Voznja? voznja;
  DateTime? datumKreiranja;

  Recenzija(
    this.id,
    this.ocjena,
    this.komentar,
    this.korisnik,
    this.voznja,
    this.datumKreiranja,
  );

  factory Recenzija.fromJson(Map<String, dynamic> json) =>
      _$RecenzijaFromJson(json);

  Map<String, dynamic> toJson() => _$RecenzijaToJson(this);
}
